SpatialPooler.cu

```diff
@@ -172,28 +172,6 @@ void generatePermanences(Real* per_dev, size_t per_dev_pitch, Real connectedPct,
 													synPermConnected * (Real)curand_uniform(&localState);
 }
 
-__device__
-void calculateOverlap(bool* in_dev, volatile bool* in_sh, UInt* pot_dev, Real* per_dev, Real* boosts_dev, UInt* numPot_dev, volatile UInt* olaps_sh, Real threshold, const UInt inBlockSize, const UInt MAX_CONNECTED)
```

Comment:
If it's possible, leave all the (SP related) code here, and remove from HelloSP. The intention is to have SP implemented in SpatialPooler.cu and just call init & compute() from the other file.

Reply:
SpatialPooler.cu is for device code. What you call "duplicit" code is HelloSP.cu is not duplicated at all, it can be used to initialize permanences and potential pools on host. This code is not used now and I can delete it if it bothers your eyes...

Reply:
OK, I have removed it.